fw/client.c

```diff
@@ -292,7 +374,7 @@ tfw_client_start(void)
 	if (!client_db)
 		return -EINVAL;
 
-	return 0;
+	return tfw_client_init_lru();
```

Comment:
There is a leak here! If `tfw_client_init_lru` fails `tfw_client_stop` will never called (just try to return -ENOMEM in `tfw_client_init_lru` instead of memory allocation). And tdb_close(client_db); will never called